src/reporter/api.py

```diff
@@ -0,0 +1,21 @@
+
+def list_of_api():
+    return {
```

Comment:
I also agree with @chicco785. Also, unless we can automatically generate this file from the API spec, paths are bound to get out of sync since it's easy to change a path in one place and forget to do it in the other. I'd rather avoid duplicating paths if possible :-)

Reply:
@chicco785 , I understand your concern. But as per @taliaga comment, we considered to list down all the available API's. So, if you can please confirm which API needs to be added to proceed further?

Reply:
sorry, my message was not clear, i meant that we don't need to explode the apis:

```
{
'config_url': '/v2/config',
'health_url': '/v2/health',
'notify_url': '/v2/notify',
'subscriptions_url': '/v2/subscriptions',
'entities_url': '/v2/entities',
'types_url': '/v2/types',
'attributes_url': '/v2/attrs'
}
```